invenio_cli/commands/packages.py

```diff
@@ -17,7 +17,7 @@ class PackagesCommands(object):
     """Local installation commands."""
 
     @staticmethod
-    def install_packages(packages):
+    def install_packages(packages=None):
```

Comment:
major: why this change!!? install requires a minimum of 1 package, is different from update.